seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/sql/zeta/functions/DateTimeFunction.java

```diff
@@ -541,4 +543,16 @@ public static Integer year(List<Object> args) {
         LocalDate localDate = convertToLocalDate(datetime);
         return localDate.getYear();
     }
+
+    public static String fromUnixTime(List<Object> args) {
+        Long unixTime = (Long) args.get(0);
+        if (unixTime == null) {
+            return null;
+        }
+        String format = (String) args.get(1);
+        DateTimeFormatter df = DateTimeFormatter.ofPattern(format);
+        LocalDateTime datetime =
+                Instant.ofEpochSecond(unixTime).atZone(ZoneId.systemDefault()).toLocalDateTime();
+        return df.format(datetime);
+    }
```

Comment:
It's better to add a timezone parameter.
```
FROM_UNIXTIME(timestamp, format); // use default zone
FROM_UNIXTIME(timestamp, format, timezone); // use given timezone
```

Reply:
I've already add doc about this function in sql-functions.md
add `timezone` optional parameter in this function
and  add `timezone` case in the unit test.